### PR TITLE
Progress bar color change indicating HP range

### PR DESCRIPTION
**Pull request by giribroba**

### CODE REVIEW

**Comment by AshtrayShetty** [comment]
Hey!
This PR looks good. But, after the player loses a battle and goes to the next one, on selecting the same pokemon as in the previous battle, the HP bar of that pokemon is red (until the pokemon actually starts attacking).
Pokémon 27-10-2020 09_02_36

File: Bootleg-Pokémon/MainWindow.xaml.cs [hunk 1]
```diff
@@ -494,6 +494,8 @@ private void Item_Use_Click(object sender, RoutedEventArgs e)
 
         private void RaiseBattleMessages(object sender, GameMessageEventArgs e)
         {
+            SetHpColor(_gameSession.EnemyPokemon);
+            SetHpColor(_gameSession.CurrentPlayer.ChosenPokemon);
```
Comment (AshtrayShetty) [inline Bootleg-Pokémon/MainWindow.xaml.cs:498]: I believe these could be put under the `Move_Click()` function (starts at line 277) as the change occurs after the pokemon attacks.

**Review by AshtrayShetty — CHANGES_REQUESTED** [review]

File: Bootleg-Pokémon/MainWindow.xaml.cs (hunk 1 above)
Comment (giribroba) [reply Bootleg-Pokémon/MainWindow.xaml.cs:498]: Ok, I will make the changes and try again

Comment (giribroba) [reply Bootleg-Pokémon/MainWindow.xaml.cs:501]: > I believe these could be put under the `Move_Click()` function (starts at line 277) as the change occurs after the pokemon attacks.

I try it, but don't work very well. Anyway I make some changes and it's working.

Comment (AshtrayShetty) [reply Bootleg-Pokémon/MainWindow.xaml.cs:501]: The issue still persists, this time for the enemy pokemon.
Pokémon 28-10-2020 09_05_46

**Comment by giribroba** [comment]
I think now it's working

File: Bootleg-Pokémon/MainWindow.xaml.cs (hunk 1 above)
Comment (AshtrayShetty) [reply Bootleg-Pokémon/MainWindow.xaml.cs]: Everything seems to be resolved now. Good job!!!

**Pull request merged**